pkg/conc/ddb_lock_provider_test.go

```diff
@@ -23,15 +23,34 @@ type ddbLockProviderTestSuite struct {
 	suite.Suite
 	ctx        context.Context
 	repo       *ddbMocks.Repository
-	clock      clockwork.FakeClock
-	backOff    *backoff.ExponentialBackOff
+	clock      clock.Clock
+	backOff    backoff.BackOff
 	uuidSource *uuidMocks.Uuid
 	provider   conc.DistributedLockProvider
 
 	resource string
 	token    string
 }
 
+type testBackOff struct {
```

Comment:
please add a factory to create one

Reply:
Do I really need to add a proper factory for some one-off code in a unit test? If I make this a proper module and everything, I would need to put it somewhere other code can reuse it - do you have a place for that? If we don't want to do that, I would just keep it as it is (I am also not adding factory functions to test cases I write even though they are often also represented as a struct).